crates/annuity/src/lib.rs

```diff
@@ -108,7 +108,7 @@ pub mod pallet {
         pub fn withdraw_rewards(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
             let dest = ensure_signed(origin)?;
             let value = T::BlockRewardProvider::withdraw_reward(&dest)?;
-            let _ = T::Currency::transfer(&Self::account_id(), &dest, value, ExistenceRequirement::KeepAlive);
+            T::Currency::transfer(&Self::account_id(), &dest, value, ExistenceRequirement::AllowDeath)?;
```

Comment:
Also changed this to `AllowDeath` for benchmark tests, but since we don't have ED anyway I think this is fine